activation/handler.go

```diff
@@ -153,12 +169,24 @@ func (h *Handler) ProcessAtx(ctx context.Context, atx *types.VerifiedActivationT
 //   - ATX LayerID is NIPostLayerTime or less after the PositioningATX LayerID.
 //   - The ATX view of the previous epoch contains ActiveSetSize activations.
 func (h *Handler) SyntacticallyValidateAtx(ctx context.Context, atx *types.ActivationTx) (*types.VerifiedActivationTx, error) {
+	var (
+		commitmentATX *types.ATXID
+		err           error
+	)
+
 	if atx.PrevATXID == *types.EmptyATXID {
 		if err := h.validateInitialAtx(ctx, atx); err != nil {
 			return nil, err
 		}
+		commitmentATX = atx.CommitmentATX
```

Comment:
where is atx.CommitmentATX checked for nil in this case? i can't seem to find it in the code...

Reply:
`h.validateInitialAtx` returns an `error` if `atx.CommitmentATX` does not reference an existing valid ATX.